wagtail_ab_testing/static_src/main.tsx

```diff
@@ -36,4 +39,20 @@ document.addEventListener('DOMContentLoaded', () => {
             }
         });
     });
+
+    // A/B testing tab on page edito
+    if (abTestingTabProps) {
+        $('ul.tab-nav').append(`<li role="tab" aria-controls="tab-abtesting">
+            <a href="#tab-abtesting" class="">${gettext('A/B testing')}</a>
+        </li>`);
+        $('div.tab-content').append(`
```

Comment:
If we're loading jQuery into the final js code, I'm wondering if we could either make more use of it or to convert this to vanilla JS. Thoughts?

Reply:
Yep, I agree with not using jQuery. This could probably be replaced with ``innerHTML = ``